Tasks/DotNetCoreCLIV2/task.json

```diff
@@ -17,7 +17,7 @@
     "demands": [],
     "version": {
         "Major": 2,
-        "Minor": 210,
+        "Minor": 214,
```

Comment:
What caused this to bump beyond 211?

Reply:
the version is bumped with the sprint see https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md